### PR TITLE
feat #90: performance-specifications block v2

### LINKED ISSUE #90: FR: Performance Specifications
**February 21, 2024 - updated description:** 
After much back and forth with the engineering product marketing team and digital marketing teams, we have decided the best MVP that meet's each team's requirements will be to remove all Power data from the engine graphs to only display the Torque data, which is provided in the attached excel sheets. The [Figma Performance Specs graph designs ](https://www.figma.com/file/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?type=design&node-id=2243%3A138004&mode=dev)should still be used, with the only change in only displaying the Torque data instead of both the Torque and Power.

**Example:** 
Image

**Most up to date dev preview links:**
[MP7](https://main--vg-macktrucks-com--hlxsites.hlx.page/drafts/shomps/performance-specifications/new-charts/mp7/)
[MP8](https://main--vg-macktrucks-com--hlxsites.hlx.page/drafts/shomps/performance-specifications/new-charts/mp8/)
[MP8HE](https://main--vg-macktrucks-com--hlxsites.hlx.page/drafts/shomps/performance-specifications/new-charts/mp8he/)

Location of chart storage in SharePoint (This is old. Will need to be replaced with newest excel sheets below that remove Power data):
[MP7 Chart](https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B1BCDD186-3552-4B1D-88B0-8428F86E35CE%7D&file=performance-specifications.xlsx&action=default&mobileredirect=true)
[MP8 Chart](https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7BACCBE76E-D09B-47B5-9378-4424247C6034%7D&file=performance-specifications.xlsx&action=default&mobileredirect=true)
[MP8HE Chart](https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7BDA658340-1E01-4A79-BB4F-FB11543D4F71%7D&file=performance-specifications.xlsx&action=default&mobileredirect=true)

**New Engine Performance Spec Graph Data:** 
[mp8-performance-specifications_240220.xlsx](https://github.com/hlxsites/vg-macktrucks-com/files/14362787/mp8-performance-specifications_240220.xlsx)
[mp8he-performance-specifications_240220.xlsx](https://github.com/hlxsites/vg-macktrucks-com/files/14362786/mp8he-performance-specifications_240220.xlsx)
[mp7-performance-specifications_240220.xlsx](https://github.com/hlxsites/vg-macktrucks-com/files/14362788/mp7-performance-specifications_240220.xlsx)

The following description is to be used as a placeholder for engine descriptions. @WendyKruger will update the descriptions as needed. 
image

**Requirements** 

- As a content editor I can add and change the engine descriptions as needed 
- Remove all Power data from the engine graphs to only display the Torque data, which is provided in the attached excel sheets. The [Figma Performance Specs graph designs ](https://www.figma.com/file/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?type=design&node-id=2243%3A138004&mode=dev)should still be used, with the only change in only displaying the Torque data instead of both the Torque and Power.
- The same PDF per engine when selecting button "download"

**Design** 

image

**Notes**

- Can reuse tab carousel block
  - navigation on mobile will work the same as existing tabs carousel
- Maximum number of tabs needed is 5

### PR DESCRIPTION
**Pull request by synox**
Fix #90, using the new design in Figma. 

-> [Excel](https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/Shared%20Documents/sites/VolvoGroup/vg-macktrucks-com/drafts/wingeier/performance-specifications-mp8.xlsx?d=we6a7cdaa3b564771afc7b0711d417f8d&csf=1&web=1&e=Bn8XEQ)

Word structure: 
Screenshot 2023-06-15 at 18 04 02

Test URLs:
- After: https://90-performance-specifications-v2--vg-macktrucks-com--hlxsites.hlx.live/drafts/wingeier/performance-specifications
- https://90-performance-specifications-v2--vg-macktrucks-com--hlxsites.hlx.page/block-library/blocks/performance-specifications

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -120,6 +120,7 @@
   --body-font-size-s: 1.6rem; /* 16px */
   --body-font-size-xs: 1.4rem; /* 14px */
   --body-font-size-xxs: 1.3rem; /* 13px */
+  --body-font-size-xxxs: 1.1rem; /* 11px */
```
Comment (synox) [inline styles/styles.css:123]:
```suggestion
```

File: styles/styles.css [hunk 2]
```diff
@@ -559,6 +564,10 @@ main .section.magazine-listing-content h3 {
   margin: 0 0 13px;
 }
 
+main sup {
+  font-size: clamp(var(--body-font-size-xxxs), 12px, 1em);
```
Comment (synox) [inline styles/styles.css:568]:
```suggestion
  font-size: clamp(var(--body-font-size-xxs), 12px, 1em);
```

**Review by jonatan-lledo-netcentric — APPROVED** [review]

File: blocks/performance-specifications/performance-specifications.js [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+      fontFamily: 'Helvetica Neue LT Pro 55 Roman',
+      itemHeight: 25,
+      itemGap: 10,
+      textStyle: {
+        fontSize: 15,
+        color: '#ffffff',
+      },
+    },
+
+    series: getEchartsSeries(1300, 1700),
```
Comment (synox) [inline blocks/performance-specifications/performance-specifications.js:285]: Can one of the fields (Peak Power, Peak Power RPM , Peak Torque , 
Peak Torque RPM ) be used to set the gray background rectangle? Or is this an additional field? And wat is the technical term for that field? 
Screenshot 2023-06-19 at 10 07 29

@jonatan-lledo-netcentric @manuel-vara @Joesturbo

Comment (manuel-vara) [reply blocks/performance-specifications/performance-specifications.js:285]: @ZoeBeykirch , @Kannan-CM  do we have perhaps an answer for @synox here above? :)

**Pull request merged**